workspace-manager-clienttests/README.md

```diff
@@ -42,13 +42,39 @@ See `test-runner-integration.yml` and `test-runner-nightly-perf.yml` in `.github
 
 Integration Test
 ```
-./gradlew  runTest --args="suites/BasicIntegration.json /tmp/TR"
+./gradlew runTest --args="suites/BasicIntegration.json /tmp/TR"
 ```
 
 Perf Test
 ```
-./gradlew  runTest --args="suites/BasicPerf.json /tmp/TR"
+./gradlew runTest --args="suites/BasicPerf.json /tmp/TR"
+```
+
+#### Run resiliency test
+To run a resiliency test case, specify the path of the test configuration under the `suites` or `configs` directory as shown in the following examples.
+In order to run resiliency tests, an in-cluster namespaced Service Account with special RBAC role is required.
+Please refer to document on Test Runner Service Accounts for details:
+https://docs.google.com/document/d/1-fGZqtwEUVRMmfeZfUVrn2V3M2D_eonNLepuX0ve6nM/edit?usp=sharing
+ 
+
+Resiliency Test
+
+The first argument to the `runTest` task is the location of the resiliency test configs under the `src/main/resources` directory.
+The second argument is the directory to put the test results in.
+
 ```
+./gradlew runTest --args="suites/BasicResilency.json /tmp/TR"
+./gradlew runTest --args="configs/resiliency/DeleteInitialPods.json /tmp/TR"
+```
+
+#### Script for rendering Kubernetes Service Account Secrets
+
+Please use the script `render-k8s-config.sh` with a valid namespace to render the in-cluster service account secrets for resiliency tests.
+Examples of valid namespaces are `ichang`, `zloery`, etc.
```

Comment:
The [Test Runner Service Accounts doc](https://docs.google.com/document/d/1-fGZqtwEUVRMmfeZfUVrn2V3M2D_eonNLepuX0ve6nM) provides the steps to Terraform SA for all namespaces. Currently `ichang` is the only namespace with manually created SA following the steps provided in the document. The document provides `devOps` with sufficient details to Terraform SAs for other namespaces, although anyone with the right `kubectl` permissions can follow the document to provision the SAs in their own namespace.

Reply:
Hmm ok perhaps don't include `zloery` in the list of valid namespaces here then.

Bigger picture though, it feels like we need a more central place to run/demo these things, so that your personal environment is not the only place folks can try this out. Probably good to talk to WSM folks for any suggestions (`wsmtest` maybe? that may conflict with existing integration tests though)

Reply:
The resiliency test now targets `wsmtest`. I set up the SAs in `wsmtest` following the steps in the [Test Runner Service Accounts ](https://docs.google.com/document/d/1-fGZqtwEUVRMmfeZfUVrn2V3M2D_eonNLepuX0ve6nM) doc. Probably the best way to get folks familiar with the resiliency test process is for me to write a template / script to generate the SAs in any namespace and run resiliency tests against it.

Reply:
That sounds very helpful. Not sure whether there's a definitive guide somewhere to setting up a personal environment, but would be good to link your instructions/script there.

Also, I think you can swap `zloery` with `wsmtest` if that one is already setup correctly.

Reply:
Added a bunch of scripts that basically is a rundown of the steps (automated) to set up a namespace for running resiliency tests. Take a look at `setup-k8s-testrunner.sh` and the templates referenced in the script.